### PR TITLE
Peter/header

### PR DESCRIPTION
**Pull request by PeterFreemann**
header and logo components

### CODE REVIEW

File: lagcloud-website/src/Test.jsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+function Test() {
```
Comment (kobomarun) [inline lagcloud-website/src/Test.jsx:1]: delete this file

File: lagcloud-website/src/index.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.css';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
```
Comment (kobomarun) [inline lagcloud-website/src/index.js:5]: remove this liine

File: lagcloud-website/src/index.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+root.render(
+  <React.StrictMode>
+    <App />
+  </React.StrictMode>
+);
+
+// If you want to start measuring performance in your app, pass a function
+// to log results (for example: reportWebVitals(console.log))
+// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
+reportWebVitals();
```
Comment (kobomarun) [inline lagcloud-website/src/index.js:17]: remove this line

File: lagcloud-website/src/Components/Logo.js [hunk 4]
```diff
@@ -0,0 +1,6 @@
+function Logo(){
+    return(
+        <a className='navbar-brand' href='#'><img src='static/images/lagcloud-removebg-preview.png' alt='logo'/></a>
```
Comment (kobomarun) [inline lagcloud-website/src/Components/Logo.js:3]: make this line look like the code below
<a className='navbar-brand' href='#'>
    logo
</a>

File: lagcloud-website/src/App.css [hunk 5]
```diff
@@ -0,0 +1,41 @@
+.App {
```
Comment (kobomarun) [inline lagcloud-website/src/App.css:1]: remove this file

**Review by kobomarun — CHANGES_REQUESTED** [review]
Kindly made changes based on the comments made

File: lagcloud-website/src/Test.jsx (hunk 1 above)
Comment (PeterFreemann) [reply lagcloud-website/src/Test.jsx:1]: done @kobomarun

File: lagcloud-website/src/App.js
Comment (fodeyemi) [inline lagcloud-website/src/App.js:1]: If this is a React component it should be a `.jsx` file, not `.js`

File: lagcloud-website/src/App.test.js
Comment (fodeyemi) [inline lagcloud-website/src/App.test.js:1]: React component should be in .jsx files please, discussed with @PeterFreemann

**Pull request merged**